src/Application/Checker/ProductVariantLowestPriceDisplayChecker.php

```diff
@@ -48,9 +48,10 @@ private function areAllTaxonsOfProductExcluded(ProductInterface $product, Channe
             $channel->getTaxonsExcludedFromShowingLowestPrice()->toArray(),
```

Comment:
early return when the channel has no excluded taxons

Reply:
As a side note, the `getExcludedTaxonsWithChildren` method needs to be refactored, with a complicated enough taxon structure the performance will plummet, and since now we don't care if all product's taxons are excluded we can just return immediately when any match is found.